### ISSUE TITLE
Optionally display destination info

### ISSUE THREAD

**Issue by thisconnect** [ritz078/rollup-plugin-filesize#12]
When multiple bundles are generated it would be great to have the option to display which bundle the size information relates to. I.E

```
Dest: dist/scripts/bundle.js
Bundle size: 4842 B, Gzipped size: 1669 B
```

**Comment by Havunen** [comment]
I was looking for same feature :)

**Comment by ritz078** [comment]
available in v1.2.0

**Comment by thisconnect** [comment]
hi @ritz078  thanks a lot!!!
